LocalConfig.pri.orig

```diff
@@ -51,4 +51,3 @@ QMAKE_CXXFLAGS+=-Werror
                          
 # Use this with valgrind
 #QMAKE_CXXFLAGS+=-g -O1 
-
```

Comment:
Sometimes removing the last empty line in a file can cause some issues.  No changes are needed to this file.